robocopkaka/bootcamp-29-project
# #157770845 Clean up state and reducers

#### What does this PR do?
make modifications to initial state and reducers
#### Description of Task to be completed?
* remove redundant action types
* move the center object in state to the centers object
* move the event object in state to the events object
* delete client/reducers/singleEventReducer.js
* delete client/reducers/singleCenterReducer.js
* remove center and event from the root reducer
* move case for fetching a single event to client/reducers/eventReducer.js
* move case for fetching a single center to client/reducers/centerReducer.js
* modify necessary components to take advantage of the new shape of state
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8080` on your browser
* install the redux extension for your browser
* `inspect` or open your browser's dev tools
* click on `Redux`
* click on `State`
* you shouldn't see a `center` and an `event object`
#### Any background context you want to provide?
state operations were becoming a bit complicated since I was using redundant methods
#### What are the relevant pivotal tracker stories?(if applicable)
#157770845
